tests/HelpSelfTest.elm

```diff
@@ -21,14 +21,7 @@ suite =
                 view {}
                     |> Html.Styled.toUnstyled
                     |> Query.fromHtml
-                    |> Query.contains [ Html.text "Help Self page - grid and singles" ]
-        , test "HelpSelf view has 3 nav links" <|
```

Comment:
Did you deliberately remove the counting links test?

Reply:
Yes, I wasn't sure it was helpful to count the number of links at this stage. I made five blocks mostly for responsive styling, but expect that to change.

Reply:
Agree.